test/unit/alphabet/composite/alphabet_variant_test.cpp

```diff
@@ -332,3 +333,31 @@ TEST(alphabet_variant_test, two_different_variants)
     seqan3::alphabet_variant<seqan3::rna4, seqan3::gap> r2{'A'_rna4};
     EXPECT_EQ(l, r2); // this works because rna4 and dna4 are implicitly convertible to each other
 }
+
+TEST(alphabet_variant_test, char_is_valid_for)
+{
+    using char_t = seqan3::alphabet_char_t<seqan3::gapped<seqan3::rna5>>;
+
+    char_t i = std::numeric_limits<char_t>::min();
+    char_t j = std::numeric_limits<char_t>::max();
+
+    for (; i < j; ++i)
+    {
+        EXPECT_EQ(seqan3::char_is_valid_for<seqan3::gapped<seqan3::rna5>>(i),
+                  seqan3::char_is_valid_for<seqan3::gap>(i) || seqan3::char_is_valid_for<seqan3::rna5>(i));
+    }
+}
+
+TEST(alphabet_variant_test, is_in_alphabet)
```

Comment:
Move me into `char_predicate_test`.

Also have a second test that tests the specific equivalent classes.

Reply:
> Move me into char_predicate_test.

See https://github.com/seqan/seqan3/pull/1973#discussion_r452115051

> Also have a second test that tests the specific equivalent classes.

What do you mean by that?

Reply:
This is a regression test for `seqan3::is_in_alphabet` and that lives in `test/unit/core/char_operations/char_predicate_test.cpp`